server/lib/nl/detection/main.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Endpoints for Datacommons NL"""
+
+import logging
+from typing import Dict
+
+from flask import current_app
+
+import server.lib.nl.common.counters as ctr
+from server.lib.nl.detection.place import get_place_from_dcids
+from server.lib.nl.detection.place import infer_place_dcids
+from server.lib.nl.detection.place import remove_places
+from server.lib.nl.detection.types import ClassificationType
+from server.lib.nl.detection.types import Detection
+from server.lib.nl.detection.types import NLClassifier
+from server.lib.nl.detection.types import PlaceDetection
+from server.lib.nl.detection.types import SimpleClassificationAttributes
+from server.lib.nl.detection.types import SVDetection
+
+
+def _empty_svs_score_dict():
```

Comment:
nit: same comment about `main.py` as under the config_builder/main.py

Reply:
renamed